### DISCUSSION TITLE
Need help refreshing website as wallpaper

### DISCUSSION THREAD

**Discussion by bh00per** [rocksdanister/lively#1866] [Q&A] [answered]
First of all - GREAT SOFTWARE!  Love the options it provides!

Now, my question - I use Lively to display a page from my website as the wallpaper.  It is in on a large screen TV in a common room via a mini PC running Windows 11, and the webpage I use as wallpaper is itself a collection of other webpages (our calendar, a couple of informational spreadsheets, local weather, memo board, etc. - all hosted on my website as 'www.mywebsite.com/wall display')

Used to just keep a Chrome tab open fullscreen, but of course anytime some opened a new tab or any other app it was never returned to full screen or was closed out instead.  This, of course, is how I found Lively as an option to display a website as the wallpaper.

However, the webpage I host updates dynamically as I add/remove information.  The wallpaper version of the webpage does not seem to. (IE: The current day on the calendar is always highlighted, and if I go away for days it will still be highlighted days earlier, and no new info will be reflected on the wallpaper)

If I go to the system tray icon and click "Change Wallpaper", it updates immediately to whatever I have changed on 'www.mywebsite.com/walldisplay'

I've searched the settings and cannot seem to find a way to set a refresh time.  Is there a setting I can use to do so?  (just to have Lively refresh the website I have entered into the wallpaper settings)

If not, what might be my best option for a script to effect the same change I get when I manually click "change wallpaper" in the system tray icon?

TLDR - set a website as my wallpaper, want to make it refresh/update periodically (every 30 minutes or 1 hour, etc)

Thanks in advance!

**Comment by rocksdanister** [comment, marked as answer]
Just set the same wallpaper again:
https://github.com/rocksdanister/lively/wiki/Command-Line-Controls#set-wallpaper
